src/components/shared/TreeView.css

```diff
@@ -11,9 +11,20 @@
   user-select: none;
 }
 
+.treeViewRow,
 .treeViewHeader {
-  position: relative;
-  height: 16px;
+  display: flex;
+  flex-flow: row nowrap;
+  justify-content: flex-start;
+  white-space: nowrap;
+}
+
+.treeViewRowScrolledColumns {
+  align-items: center;
+}
```

Comment:
I went back and forth with that.

The problem: with `center` for all columns, then we don't see a border anymore for the cells that are empty. So I kept the default `stretch` for them. But we definitely want `center` for the scrolled columns, because it has various elements in int and we want them to be centered. Another possibility would be adding another wrapping element, but I didn't want to do that just yet (maybe we'll have to in the future).

Reply:
Hm, I'm okay with the current way of doing the scrollable ones center and static ones stretch. It would be better if we can make it more generic later but I wouldn't spend too much time on it for now.